src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -418,22 +423,32 @@ private extension RiskConsumer {
 extension RiskProvider {
 	private func _requestRiskLevel_Mock(userInitiated: Bool) {
 		let risk = Risk.mocked
-		successOnTargetQueue(risk: risk)
 
-		for consumer in consumers {
-			_provideRiskResult(.success(risk), to: consumer)
+		switch risk.level {
+		case .high:
+			store.riskCalculationResult = RiskCalculationResult(
+				riskLevel: .high,
+				minimumDistinctEncountersWithLowRisk: 0,
+				minimumDistinctEncountersWithHighRisk: 0,
+				mostRecentDateWithLowRisk: risk.details.mostRecentDateWithRiskLevel,
+				mostRecentDateWithHighRisk: risk.details.mostRecentDateWithRiskLevel,
+				numberOfDaysWithLowRisk: risk.details.numberOfDaysWithRiskLevel,
+				numberOfDaysWithHighRisk: risk.details.numberOfDaysWithRiskLevel,
+				calculationDate: Date()
+			)
+		default:
+			store.riskCalculationResult = RiskCalculationResult(
+				riskLevel: .low,
+				minimumDistinctEncountersWithLowRisk: 0,
+				minimumDistinctEncountersWithHighRisk: 0,
+				mostRecentDateWithLowRisk: nil, //risk.details.mostRecentDateWithRiskLevel,
```

Comment:
As I remember by setting this value to nil, the logic will run into an assert.
Could we just set it to "risk.details.mostRecentDateWithRiskLevel" ?

Reply:
fixed with commit 930d3cd194eb6786b356bb6f08e00c1dc9ba1afa